charts/portworx/templates/hooks/post-install/px-create-cluster-token.yaml

```diff
@@ -1,35 +1,37 @@
-{{- if (.Values.clusterToken.create) }}
-  {{- $customRegistryURL := .Values.customRegistryURL | default "none" }}
-  {{- $registrySecret := .Values.registrySecret | default "none" }}
-
+{{- if .Values.clusterToken.create }}
+{{- $customRegistryURL := .Values.customRegistryURL | default "null" }}
+{{- $registrySecret := .Values.registrySecret | default "null" }}
 apiVersion: batch/v1
 kind: Job
 metadata:
+<<<<<<< HEAD
```

Comment:
This needs a relook. The merge markers are still there.

Reply:
Resolved.